packages/ubuntu_desktop_installer/lib/l10n/app_en_US.arb

```diff
@@ -48,6 +48,16 @@
   "pressOneKey": "Please press one of the following keys:",
   "isKeyPresent": "Is the following key present on your keyboard?",
 
+  "turnOffSecureBootTitle": "Turn off secure boot",
```

Comment:
sorry, the title is wrong in the screenshot. in the [spec](https://people.canonical.com/~platform/desktop/installer/7.%20Turn%20Off%20Secure%20Boot/) it says "Configure secure boot" which is the correct one (following what ubiquity does, and this is more complex than flipping a switch anyway)